spec/factories/messages.rb

```diff
@@ -0,0 +1,12 @@
+require 'faker'
+
+FactoryGirl.define do
+  factory :message do |m|
+    m.content {Faker::Lorem.sentence(5)}
+    m.user_id '1'
```

Comment:
Also, it's better to provide a user to the message when you create it as a factory. I would do some digging into FactoryGirl and see how it can work with associations.